src/core/types/decimal192.rs

```diff
@@ -19,35 +19,6 @@ impl crate::UniffiCustomTypeConverter for InnerDecimal {
     }
 }
 
-/// A 192 bit precision Decimal wrapping Scrypto's `Decimal` type, but
-/// giving it UniFFI conversion.
-///
-/// The purpose of this "Inner" Decimal is ensuring that the `Decimal` type
-/// is not exported as a `String` in FFI land (Swift/Kotlin). The current
-/// design ensure that `Decimal` is converted into a Swift `struct` / Kotlin
-/// `data class` that **has** a `inner: String`
-#[derive(
-    Clone,
-    Copy,
-    Debug,
-    PartialEq,
-    Eq,
-    PartialOrd,
-    Ord,
-    Hash,
-    derive_more::Display,
-)]
-pub struct InnerDecimal(pub(crate) ScryptoDecimal192);
```

Comment:
I had this because I (incorrectly) thought we could not add custom UniFFI conversion for external types, but we can, so no need for this level of indirection.